presto-orc/src/main/java/com/facebook/presto/orc/TestingEncryptionLibrary.java

```diff
@@ -35,7 +35,7 @@
     @Override
     public byte[] encryptKey(byte[] keyMetadata, byte[] input, int offset, int length)
     {
-        return encrypt(keyMetadata, input, offset, length);
+        return keyMetadata;
```

Comment:
Could you please explain why we needed this change?

Reply:
I guess this was because the key was not encrypted in the manually created dwrf file such that decrypting it would fail while reading. However, @rschlussel does this affect other tests in `TestDecryption`? The test coverage maybe reduced if `encryptKey` and `decryptKey` were not tested. Would it make sense to make a new `TestingPlainKeyEncryptionLibrary` that is a subclass of `TestingEncryptionLibrary`? 
If you think this is not the case or can be fixed later it's also ok.

Reply:
That's fair. I changed it because I figured the loss in test coverage would be minimal, so why have 2 classes, but I think it is still some loss in coverage, so it makes sense to keep it as it was and introduce a new class.